site/_posts/2023-02-03-single-sign-on-laravel-fusionauth.md

```diff
@@ -0,0 +1,388 @@
+without adding any complexity to your application.
+
+In this tutorial, you'll learn how to design and implement SSO using [Laravel](https://laravel.com), a popular PHP-based
+web framework and FusionAuth as the OIDC provider. Any other OIDC compatible authentication server should work as well.
+
+## Implementing SSO in a Laravel Web App
+
+As previously stated, in this tutorial, you'll be shown how to implement SSO in a Laravel web app. The Laravel demo
+application is integrated with FusionAuth, an authentication and authorization platform,
+and [Laravel Socialite](https://laravel.com/docs/9.x/socialite), its official solution to authenticate users with OAuth
+providers.
+
+Before you begin, you'll need the following:
+
+* A Linux machine. The step-by-step instructions in this article are based on
+  a [CentOS Linux machine](https://www.centos.org). If you want to work on a different OS, check out
+  this [setup guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide) for more information.
+* [Docker Engine](https://docs.docker.com/engine/) and [Docker Compose](https://docs.docker.com/compose/).
+* Experience with a Laravel framework and application development.
+
+### Installing FusionAuth
+
+Go through [5-Minute Setup Guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide) to install FusionAuth in your
+machine. You have a few ways of doing so, but we recommend
+the [Docker](https://fusionauth.io/docs/v1/tech/getting-started/5-minute-docker) approach.
+
+#### Starting FusionAuth
+
+{% include posts/sso/starting.md %}
+
+#### Configure FusionAuth
+
+Now you need to configure FusionAuth.
+
+First, you'll configure the application, then register the user for the application.
+
+This setup allows users in FusionAuth to sign in to the Laravel application automatically once they are authenticated by
+FusionAuth.
+
+#### Set up the application
+
+{% include posts/sso/setup.md callbackUrl='http://localhost/auth/callback' oauthDetailsImage='/assets/img/blogs/laravel-single-sign-on/oauth-details.png' %}
+
+#### Register the user
+
+{% include posts/sso/register-user.md %}
+
+#### Kickstart
+
+Instead of manually setting up FusionAuth using the admin UI as you did above, you can use Kickstart. This tool allows
+you to get going quickly if you have a fresh installation of FusionAuth. Learn more about how to
+use [Kickstart](https://fusionauth.io/docs/v1/tech/installation-guide/kickstart).
+
+> @TODO kickstart json file
+
+### Installing Laravel
+
+> Note: If you already have a running Laravel application, please skip this step.
+
+There are several ways of installing Laravel, but we recommend using it inside a Docker container via Laravel Sail. To
+do so, you can execute the command below to automatically download and install every package needed:
+
+```shell
+$ curl -s https://laravel.build/my-app | bash
+```
+
+> Hint: you can change the name of your application from `my-app` from the URL above to anything you like, as long
```

Comment:
I like the hints, but let's use the callouts documented in https://github.com/FusionAuth/fusionauth-site/blob/master/DocsDevREADME.md in the section:

"When writing blog posts, you have access to callouts"